apps/api/src/app/workflows-v2/exceptions/workflow-already-exist.ts

```diff
@@ -0,0 +1,12 @@
+import { BadRequestException } from '@nestjs/common';
+import { UpsertWorkflowCommand } from '../usecases/upsert-workflow/upsert-workflow.command';
+
+export class WorkflowAlreadyExistException extends BadRequestException {
+  constructor(command: UpsertWorkflowCommand) {
+    super({
+      message: 'Workflow with the same name already exists',
+      userExternalId: command.workflowDto.name,
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
❓ **question:** The `userExternalId` (user id) is mapping to the workflow name property, the value doesn't align with the expected name. Do we need to rename one of these?‏

Reply:
it's in an exception setting, wanted it to be the most clear  but I will change it to workflow name, maybe it's better